libpromises/mod_common.c

```diff
@@ -259,6 +259,7 @@ const ConstraintSyntax CFG_CONTROLBODY[COMMON_CONTROL_MAX + 1] =
     ConstraintSyntaxNewContextList("site_classes", "A list of classes that will represent geographical site locations for hosts. These should be defined elsewhere in the configuration in a classes promise.", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewString("syslog_host", CF_IPRANGE, "The name or address of a host to which syslog messages should be sent directly by UDP. Default value: localhost", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewInt("syslog_port", CF_VALRANGE, "The port number of a UDP syslog service. Default value: 514", SYNTAX_STATUS_NORMAL),
+    ConstraintSyntaxNewString("system_log_level", ".*", "Minimum system log level of messages that should be logged to system log", SYNTAX_STATUS_NORMAL),
```

Comment:
Definately I think we need some more detailed description of this in the docs for what specific strings users should expect to use.

In the windows event log I see: 
- Critical
- Error
- Warning
- Information
- Audit Success
- Audit Failure

What strings would I use on Linux for syslog?
- Emergency
- Alert
- Critical
- Error
- Warning
- Informational
- Debug
?

Also, I guess the string match is case insensitive, is that correct?

Reply:
It needs to be a string that our code recognizes as a valid log level so the same as the `--log-level` option of our binaries.

Reply:
I'll change the `.*` to the actual regex with all the log levels.